lib/Standalone/ConvertXsmmToFunc.cpp

```diff
@@ -96,8 +96,15 @@ struct ConvertTernaryXsmmOp : public OpRewritePattern<TernaryOp> {
 
   LogicalResult matchAndRewrite(TernaryOp ternaryOp,
                                 PatternRewriter &rewriter) const override {
+    std::string funcName =
+        "xsmm_" + stringifyEnum(ternaryOp.getCallee()).str() + "invoke";
+    if (ternaryOp.getOperandType() == "bf16")
+      funcName = funcName + "_bf16";
+    else
+      assert(ternaryOp.getOperandType() == "f32";
```

Comment:
missing a parenthesis and a message.

Reply:
Why for bf16 we add a `_bf16` while for f32 only `f32` should we add `_f32`? (to be consistent)

Reply:
do you mean the xsmm call should have an _f32 suffix? The above is just a type check, and there are no underscores. Only when composing the function call name, we are appending _bf16.

Reply:
Yeah, I would make a _f32 suffix too. Just to be consistent.